policy/modules/services/tuned.te

```diff
@@ -24,15 +45,19 @@ logging_log_file(tuned_log_t)
 type tuned_runtime_t alias tuned_var_run_t;
 files_runtime_file(tuned_runtime_t)
 
+type tuned_tmp_t;
+files_tmp_file(tuned_tmp_t)
+
 ########################################
 #
 # Local policy
 #
 
-allow tuned_t self:capability { sys_admin sys_nice };
+allow tuned_t self:capability { net_admin sys_admin sys_nice sys_ptrace sys_resource };
 dontaudit tuned_t self:capability { dac_override sys_tty_config };
 allow tuned_t self:process { setsched signal };
 allow tuned_t self:fifo_file rw_fifo_file_perms;
+allow tuned_t self:netlink_kobject_uevent_socket { bind create getattr setopt };
```

Comment:
I think you also want to allow `read`, otherwise - if I remember correctly - when an event occurs the process might enter an infinite read-loop and consume 100% CPU.

Reply:
Thanks, I didn't see tuned needing read permission, but I'm not sure I tested all functionality.  I have added read permission.